.github/workflows/haskell-ci.yml

```diff
@@ -36,10 +39,10 @@ on:
       - 'cabal.project*'
 jobs:
   linux:
-    # when changing the name or adding ghc version, please don't forget to update
-    # the .mergify.yml rules at the root of the project.
```

Comment:
Did you delete this comment on purpose because it's no longer relevant?  Or did you just miss it?  Assuming it's still relevant, we should leave it there.

Reply:
I mistook it for autogenerated comment. I will move it up so it's visible - that way we don't have to edit the file too much.